websites/D/draw.io/presence.ts

```diff
@@ -0,0 +1,18 @@
+const presence = new Presence({
+	clientId: "1175537518746279956",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/1hI3nKp.png",
+		startTimestamp: Date.now(),
+	};
+	if (document.title === "draw.io")
+		presenceData.details = "Creating a new diagram";
+	else {
+		presenceData.details = "Editing a diagram";
+		if (!(await presence.getSetting<boolean>("privacy")))
+			presenceData.state = document.title.replace(/ - draw.io$/, "");
+	}
+	presence.setActivity(presenceData);
+});
```

Comment:
Low quality presence.

Reply:
@HopperElec Is there anything else you can add to this presence?

Reply:
The draw.io site does not have a home page or anything- visiting it immediately prompts you to open or create a new diagram. They have [a separate site](https://www.drawio.com/) but this seems to be aimed more towards businesses considering using them and, especially when you consider that a PreMiD user who installs this presence would already be using it, it seems unnecessary to include this site in the presence. However, if you disagree then I'd be happy to implement it.

Reply:
Got it.